src/common/perf_counters.cc

```diff
@@ -19,21 +19,17 @@
 
 using std::ostringstream;
 
-PerfCountersCollection::PerfCountersCollection(CephContext *cct)
-  : m_cct(cct),
-    m_lock("PerfCountersCollection")
+PerfCountersCollectionImpl::PerfCountersCollectionImpl()
 {
 }
 
-PerfCountersCollection::~PerfCountersCollection()
+PerfCountersCollectionImpl::~PerfCountersCollectionImpl()
 {
   clear();
 }
 
-void PerfCountersCollection::add(class PerfCounters *l)
+void PerfCountersCollectionImpl::add(class PerfCounters *l)
```

Comment:
might want to take this opportunity to remove `class` here.

Reply:
sure.